src/DynamoCoreWpf/Properties/Resources.en-US.resx

```diff
@@ -2309,4 +2309,13 @@ Uninstall the following packages: {0}?</value>
   <data name="PreferencesViewVisualSettingsTab" xml:space="preserve">
     <value>Visual Settings</value>
   </data>
+  <data name="PreferencesWindowFontSizes" xml:space="preserve">
```

Comment:
I think these strings are already defined so we should be able to reuse the existing ones, we can rename those resources strings if needed. e.g. `ScalingMediumButton` `ScalingLargeButton` to sth else

Reply:
@QilongTang  I removed the duplicated strings in the next commit:
https://github.com/DynamoDS/Dynamo/pull/11542/commits/89b81d746df5a9d76bb3597c2f49446c392abae8